service/controller/v4/version_bundle.go

```diff
@@ -55,6 +55,6 @@ func VersionBundle() versionbundle.Bundle {
 			},
 		},
 		Name:    "azure-operator",
-		Version: "1.1.0",
+		Version: "1.2.0",
```

Comment:
The existing changelogs should be replaced with something like "Add your changes here.".

Reply:
done